tests/main_app/jobs_workers/test_worker_cancellation.py

```
@@ -79,7 +79,10 @@ def side_effect(*args, **kwargs):
     )
     monkeypatch.setattr("src.main_app.jobs_workers.collect_main_files_worker.find_main_title", lambda x: "file.svg")
     monkeypatch.setattr(
-        "src.main_app.jobs_workers.collect_main_files_worker.template_service.update_template", mock_update_template
+        "src.main_app.jobs_workers.collect_main_files_worker.find_last_world_file_from_owidslidersrcs", lambda x: None
```

Ruff (0.15.2)

[warning] Unused lambda argument: `x`

(ARG005)